asa_parser/src/asa_parse.py

```diff
@@ -200,9 +200,55 @@ def show_memory_region(self):
                 memory_region.append(mem)
         return json.dumps(memory_region)
 
+    @property
```

Comment:
It's interesting that you decided to use the property decorator and make this an attribute.  I'll give you my .02 on that...

For what we're doing in this assignment, we are accessing a file which contains the output of a "show tech" Now, that file could change on disk, but, since the __init__ reads the file, if you do change the file, it won't change the contents of any class you've already created:
```
    def __init__(self, show_tech_file):
        with open(show_tech_file) as f:
            self.shtech = f.read().splitlines()
```
To me, that implies attribute more than it does method...

Now, taking this one step further, what if we changed ShowTech so that it dynamically connected to the ASA hardware each time you called get_show_section() - this is actually what I had in mind when I created this assignment - which means calling get_show_section() would become dynamic.  To me, that says method more than attribute because it becomes dynamic and performs an action on the device we're connected to.  

Probably the most correct answer is to have a get() method which retrieves the most current information (either from the file, the actual ASA device, etc) and update the class's information (which should be an attribute)
```
asa.get_current_show_tech() # method
if 'something' in asa.show_cpu_detailed: # attribute
    print('found something')

Curious to hear your thoughts.

Reply:
To be more clear, it's fine with me if you want to leave this as a property - no additional work required here.

Reply:
My apologies I did not mean to make that a property. This was a mistake on my part. I went ahead and removed the @property decorator. I believe I made this mistake because I had clicked on a yellow bulb in python and I believe I may have clicked on "Convert method to property" by mistake.